conf/zeppelin/zeppelin-env-0.8.0.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
why is this all commented out?  Why wouldn't zeppelin-env.sh not work?

Reply:
Ah, this is the Zeppelin template. I am setting everything in the xml configuration files. However, if someone wanted to modify these they could.

Reply:
Just realized that I have it coded to use the zeppelin-env.sh that exists. Only the XML needs to be different. zeppelin-env-0.8.0.sh doesn't get used. I will remove it.